packages/jest-runner/corpus.ts

```diff
@@ -44,16 +55,56 @@ export class Corpus {
 
 	private createMissingDirectories() {
 		fs.mkdirSync(this._inputsDirectory, { recursive: true });
+		fs.mkdirSync(this._corpusDirectory, { recursive: true });
 	}
 }
 
-const buildInputsDirectory = (
+const buildDirectory = (
+	testSourceFilePath: string,
+	testJestPathElements: string[],
+	addToProjectRoot = ""
+): string => {
+	const rootDirectory = buildRootDirectory(
+		testSourceFilePath,
+		addToProjectRoot
+	);
+	const safeTestJestPathElements = testJestPathElements.map(
+		replaceSpacesWithUnderscore
+	);
+	return path.join(rootDirectory, ...safeTestJestPathElements, path.sep);
+};
+
+const buildRootDirectory = (
 	testSourceFilePath: string,
-	testJestPathElements: string[]
+	addToProjectRoot: string
```

Comment:
I'd prefer to use names that start with a verb for functions/methods. Would `projectCorpusRoot` or `inProjectCorpusRoot` be better alternatives?

Reply:
done